utils/format.go

```diff
@@ -332,6 +332,30 @@ func FormatBlockStatusShort(status uint64) template.HTML {
 	}
 }
 
+func FormatBlockStatusStyle(status uint64) template.HTML {
+	if status == 0 {
+		return `<div title="This slot is still scheduled" data-toggle="tooltip" class="style-badge style-badge-single-char style-bg-neutral-1"><p class="style-status-tag-text" >S<span class="d-none d-sm-inline">cheduled</span></p></div>`
+	} else if status == 1 {
+		return `<div title="This block has been proposed" data-toggle="tooltip" class="style-badge style-badge-single-char style-bg-good"><p class="style-status-tag-text text-white" >P<span class="d-none d-sm-inline">roposed</span></p></div>`
+	} else if status == 2 {
+		return `<div title="This block has been proposed" data-toggle="tooltip" class="style-badge style-badge-single-char style-bg-bad"><p class="style-status-tag-text" >M<span class="d-none d-sm-inline">issed</span></p></div>`
```

Comment:
The text here ("This block has been proposed") is the same as the one for status == 1. Is that intended? Unfortunately I do not see what status == 1 and status == 2 means here.